### DISCUSSION TITLE
Can make the highlighted lyrics showing in the same place?

### DISCUSSION THREAD

**Discussion by Bakabot307** [SpotX-Official/SpotX#88] [Ideas]
I'm currently using OBS to capture the highlighted lyrics from Spotify. The problem I have is at the start and the end of the song, the lyrics move away from the middle. Just asking if you could make it better. Thanks

https://user-images.githubusercontent.com/60350866/180631918-4832a216-b933-4b23-9ae5-318b357d0e9b.mp4

**Comment by amd64fox** [comment]
It was possible in the old lyrics, but the new lyrics work differently.
Try installing [`Spicetify`](https://github.com/khanhas/spicetify-cli), it has the [`lyrics-plus`](https://github.com/spicetify/spicetify-cli/tree/master/CustomApps/lyrics-plus#lyrics-plus) extension which works the old style. 

Below is an example of the beginning and end of your test song:
![7](https://user-images.githubusercontent.com/62529699/180635829-2429253b-7e52-46ec-8cb7-66f04d37d8c2.gif)

You can also use SpotX and Spicetify [together](https://github.com/amd64fox/SpotX/discussions/28#discussioncomment-2389043) if needed.
